### PR TITLE
feat: added kasparek

### LINKED ISSUE #13: /kasparek
toto nam treba asap

### PR DESCRIPTION
**Pull request by SafarRadim**

### CODE REVIEW

File: frontroomsbot/bot.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ async def flip(interaction: discord.Interaction):
     await interaction.response.send_message(f"{result}")
 
 
+@tree.command(name="kasparek", description="Zjistí jakého máš kašpara", guild=guild)
+async def kasparek(interaction: discord.Interaction):
+    unit = choice(["m", "cm", "mm", "nm"])
+    result = round(uniform(0, 50), 5)
```
Comment (jkalend) [inline frontroomsbot/bot.py]: not a fan of floats for this and especially not 5 digit floats

File: frontroomsbot/bot.py [hunk 2]
```diff
@@ -48,6 +48,15 @@ async def flip(interaction: discord.Interaction):
     await interaction.response.send_message(f"{result}")
 
 
+@tree.command(name="kasparek", description="Zjistí jakého máš kašpara", guild=guild)
```
Comment (jkalend) [inline frontroomsbot/bot.py]: kasparka :objection:

File: frontroomsbot/bot.py [hunk 3]
```diff
@@ -48,6 +48,15 @@ async def flip(interaction: discord.Interaction):
...
+async def kasparek(interaction: discord.Interaction):
+    unit = choice(["m", "cm", "mm", "nm"])
+    result = round(uniform(0, 50), 5)
+
+    message = f"{result} {unit}" if unit else f"{result}"
```
Comment (jkalend) [inline frontroomsbot/bot.py]: I would default to cms

**Review by jkalend — COMMENTED** [review]
negative numbers could be fun as well

File: frontroomsbot/bot.py (hunk 1 above)
Comment (SafarRadim) [reply frontroomsbot/bot.py]: I feel like having at least 2 digit float so we have fewer exact number could be a good thing.

**Comment by SafarRadim** [comment]
> negative numbers could be fun as well

Left only positive in, negative feel, idk, weird. (What does it mean that I have -50cm???)

**Review by jkalend — APPROVED** [review]
lgtm

**Pull request merged**